### ISSUE TITLE
Httpd collector is collecting from the Netuitive API

### ISSUE THREAD

**Issue by Grumblesmith** [Netuitive/netuitive-diamond#44]
After enabling the HttpdCollector with the following config, it is collecting from localhost and from api.app.netuitive.com:
```
enabled = True
urls = localhost http://localhost:8080/server-status?auto,
```

Log entries:
```
[2018-01-22 18:44:44,843] [MainThread:HttpdCollector:DEBUG] Starting
[2018-01-22 18:44:44,844] [MainThread:HttpdCollector:DEBUG] Interval: 60.0 seconds
[2018-01-22 18:44:44,845] [MainThread:HttpdCollector:DEBUG] Max collection time: 22 seconds
[2018-01-22 18:45:06,846] [MainThread:HttpdCollector:ERROR] Error retrieving HTTPD stats for host api.app.netuitive.com:80, url '/ingest/infrastructure?':
[2018-01-22 18:45:06,847] [MainThread:HttpdCollector:ERROR] Error retrieving HTTPD stats for host localhost:8080, url '/server-status?auto': [Errno 111] Connection refused
[2018-01-22 18:45:06,847] [MainThread:HttpdCollector:DEBUG] Collection took 22001 ms
```

**Comment by yahonggu** [comment]
The commit below merges config[‘handlers’][‘NetuitiveHandler’] to collector's base config, thus Netuitive Handler api 'url' is made available for the HttpdCollector even if they are supposed to be in different name space.
https://github.com/Netuitive/netuitive-diamond/commit/dee6b4d30443f8552ec7ac1fa954eb005a90b285#diff-bd4b18c3f6346c52166b29234a8ef7dbR206

**Comment by Grumblesmith** [comment]
Fixed by #45
